app/tests/spec/models/auth_brokers/fx-sync-channel.js

```diff
@@ -0,0 +1,351 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const _ = require('underscore');
+  const { assert } = require('chai');
+  const AuthErrors = require('lib/auth-errors');
+  const FxSyncChannelAuthenticationBroker = require('models/auth_brokers/fx-sync-channel');
+  const NullChannel = require('lib/channels/null');
+  const p = require('lib/promise');
+  const sinon = require('sinon');
+  const User = require('models/user');
+  const WindowMock = require('../../../mocks/window');
+
+  describe('models/auth_brokers/fx-sync-channel', function () {
+    var account;
```

Comment:
Wonder if it makes sense to ES5ify this entire file since the diff as it as a new file. Or maybe a follow up.